app/controllers/StatsController.php

```diff
@@ -2,21 +2,48 @@
   require_once($GLOBALS['dirpre'].'controllers/Controller.php');
 
   class StatsController extends Controller {
+
+    function loadStats() {
+      $updateArray = self::update(); // contains arrays 'stats' and 'cities'
+      $nojobsArray = self::nojobs(); // contains arrays 'recruiterEmails', 'recruiterEmailsWC', and 'recruiterEmailsWJ'
+      $studentsArray = self::students(); // contains arrays 'studentConfirmedEmails', 'studentsUnconfirmedEmails', 'allStudents'
+      $missingRecruiterArray = self::missingrecruiter(); //contains array 'missingRecruiters'
+      $recruiterByDateArray = self::recruiterbydate(); //contains array 'recruiterByDate'
+      $subletsended2014Array = self::subletsended2014(); // contains array 'subletsended2014'
+      $unknownschoolsArray = self::unknownschools(); // contains array 'domain'
+      $cumulativeArray = self::cumulative(); // contains arrays 'cumulativeviews', 'cumulativeclicks'
+      $getMessageParticipantsArray = self::getMessageParticipants(); // contains array 'messageparticipants'
+
+      $toRender = array_merge($updateArray, $nojobsArray, $studentsArray,
```

Comment:
Might want to just have each merged array simply be its own key in $toRender to avoid any possibility of conflicts.